specification/synapse/resource-manager/Microsoft.Synapse/preview/2020-04-01-preview/sqlDatabase.json

```diff
@@ -0,0 +1,299 @@
+          },
+          "202": {
+            "description": "Updating the database is in progress."
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Updates a sqldatabase.": {
+            "$ref": "./examples/PatchSqlDatabase.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "SqlDatabaseProperties": {
+      "description": "The sql database's properties.",
+      "type": "object",
+      "properties": {
+        "collation": {
+          "description": "The collation of the database.",
+          "type": "string",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "maxSizeBytes": {
+          "format": "int64",
+          "description": "The max size of the database expressed in bytes.",
+          "type": "integer"
+        },
+        "databaseGuid": {
+          "format": "uuid",
+          "description": "The Guid of the database.",
+          "type": "string",
+          "readOnly": true
+        },
+        "creationDate": {
```

Comment:
Creation/LastModified data should be pulled up to the top level using the common systemdata contract.  See https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-contracts.md#system-metadata-for-all-azure-resources.  There is a common definition for systemdata in the top level types.json